hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -184,12 +187,12 @@
     }
     { name: "av_empty",
       desc: '''
-            Raised when a transaction is NACKed because the Available Buffer FIFO for OUT or SETUP transactions is empty.
+            Raised when the AV FIFO is empty and the device interface is enabled.
```

Comment:
can you remind me if the `empty` and `full` interrupts have the same property as the interrupts above? Ie if you clear the state, the interrupt may re-appear until properly handled?

Reply:
The `av_empty` and `rx_full` interrupts will re-appear until properly handled. These are interrupts for a particular condition being active, just like `pkt_received` and `pkt_sent`, so they will continue to re-assert the interrupt pin if the condition has not been cleared (or if it has occurred between clearing the condition and acknowledging the interrupt).

In the ideal case, these interrupts wouldn't separately have a bit that needs to be cleared in the block level's interrupt status register. It's superfluous, since the condition controls the assertion and we acknowledge / complete servicing the interrupt at the PLIC already. But that would require more complex tooling and configuration.

Reply:
yeah sounds good. Could we add a comment just like the other two interrupts to clarify this?
Mostly because a few of the other modules have gone out of their way to make the empty/full more event based so that it wouldn't re-trigger.

Reply:
Done.